pkg/cmd/extension/command_test.go

```diff
@@ -387,11 +387,16 @@ func TestNewCmdExtension(t *testing.T) {
 				}
 			},
 			isTTY: true,
-			askStubs: func(as *prompt.AskStubber) {
```

Comment:
I wonder if it would be feasible to have `AskStubber` implement the Prompter interface so we can continue to use the old stubs with new places that accept a Prompter? The translation could be tricky, but the benefit would be that we could keep existing tests virtually unchanged.

Reply:
I'd prefer to be done with this work and not have three versions of prompt stubbing around so I intend, perhaps foolishly, to put my head down and get this work done.